GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleCubeScript.java

```diff
@@ -157,16 +158,15 @@ public void onStep() {
         mFrontFace.getRenderData().getMaterial().setOpacity(1.0f);
         mFrontFace2.getRenderData().getMaterial().setOpacity(1.0f);
         mFrontFace3.getRenderData().getMaterial().setOpacity(1.0f);
-        GVREyePointeeHolder[] eyePointeeHolders = GVRPicker
-                .pickScene(mGVRContext.getMainScene());
-        for (GVREyePointeeHolder eyePointeeHolder : eyePointeeHolders) {
-            if (eyePointeeHolder.getOwnerObject().equals(mFrontFace)) {
+        for (GVRPickedObject pickedObject : GVRPicker.findObjects(
+                mGVRContext.getMainScene(), 0, 0, 0, 0, 0, -1.0f)) {
+            if (pickedObject.getHitObject().equals(mFrontFace)) {
```

Comment:
Just a minor concern here. Imho, I think you are somehow giving up one use case of GVREyePointeeHolder. Probably I would rather add an API in GVREyePointeeHolder and do something like 
public boolean hitObject(GVRSceneObject sceneObject){
     return getOwnerObject().equals(sceneObject);
}

just my personal opinion.